recipe/meta.yaml

```
@@ -1,37 +1,45 @@
 {% set name = "google-auth-httplib2" %}
-{% set version = "0.1.0" %}
-
+{% set version = "0.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google-auth-httplib2-{{ version }}.tar.gz
-  sha256: a07c39fd632becacd3f07718dfd6021bf396978f03ad3ce4321d060015cc30ac
+  url: https://github.com/googleapis/google-auth-library-python-httplib2/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7e714ff7a4cd6c69b76f8c91dfe177cdce49ae8bfb398b22a7c1468f3915037d
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39]
```

```suggestion
  skip: True  # [py<36]
```
https://github.com/googleapis/google-auth-library-python-httplib2/blob/v0.2.0/setup.py#L42